editor/assets/default_renderpipeline/builtin-pipeline.ts

```diff
@@ -1604,16 +1604,12 @@ export class BuiltinUiPassBuilder implements rendering.PipelinePassBuilder {
         prevRenderPass?: rendering.BasicRenderPassBuilder)
         : rendering.BasicRenderPassBuilder | undefined {
         assert(!!prevRenderPass);
-
-        let flags = rendering.SceneFlags.UI;
+        const queue = prevRenderPass
+            .addQueue(rendering.QueueHint.BLEND, 'default', 'default');
+        queue.addDraw2D(camera);
         if (cameraConfigs.enableProfiler) {
-            flags |= rendering.SceneFlags.PROFILER;
-            prevRenderPass.showStatistics = true;
```

Comment:
I think this line is still needed.

Reply:
OK，For compatibility considerations?

Reply:
If `pass.showStatistics` is false, the profiler should not be rendered for now.